src/components/ImagesCarousel.js

```diff
@@ -133,13 +140,14 @@ export const ImagesCarousel = ({ data, navigation, refreshTimeKey, aspectRatio }
           autoplay
           loop
           autoplayDelay={0}
-          autoplayInterval={4000}
+          autoplayInterval={autoplayInterval || 4000}
           containerCustomStyle={styles.center}
           onScrollIndexChanged={setCarouselImageIndex}
         />
       )}
 
       {showSliderPauseButton &&
+        !isDisturber &&
```

Comment:
please move this additional disturber check up to the `show: showSliderPauseButton = false,` line. in the rendering part we should only read if the slider pause button should be shown or not without asking ourself anything about disturbers.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/972/commits/6e8f47ee62bb5c24c8e94350732e8531ab8ef64b